Mathlib/Data/Nat/Bits.lean

```diff
@@ -122,10 +122,20 @@ lemma bit_val (b n) : bit b n = 2 * n + cond b 1 0 := by
 lemma bit_decomp (n : Nat) : bit (bodd n) (div2 n) = n :=
   (bit_val _ _).trans <| (Nat.add_comm _ _).trans <| bodd_add_div2 _
 
-/-- For a predicate `C : Nat → Sort*`, if instances can be
+theorem shiftRight_one (n) : n >>> 1 = n / 2 := rfl
+
+theorem bit_testBit_zero_shiftRight_one (n : Nat) : bit (n.testBit 0) (n >>> 1) = n := by
```

Comment:
```suggestion
@[simp]
theorem bit_testBit_zero_shiftRight_one (n : Nat) : bit (n.testBit 0) (n >>> 1) = n := by
```

Reply:
Ah, I now see that `testBit` is not simp-normal; perhaps better to just ignore this suggestion.